src/main/java/io/hydrox/inventorysummary/InventorySummaryPlugin.java

```diff
@@ -83,6 +95,7 @@ public void onConfigChanged(ConfigChanged event)
 	{
 		if (event.getGroup().equals("inventorysummary"))
 		{
+			whitelist = Text.fromCSV(config.getWhitelist());
```

Comment:
It would be better if this branch only activated if the whitelist itself updates. This should still include the `groupItems` call, since it is needed for this.

Looking at it, I'm not sure why I left in the call at all, since what I had didn't need it.